core/controllers/features.py

```diff
@@ -39,4 +39,6 @@ def get(self, exploration_id):
                 config_domain.IS_IMPROVEMENTS_TAB_ENABLED.value,
             'is_playthrough_recording_enabled':
                 exploration_id in whitelisted_exploration_ids_for_playthroughs,
+            'can_add_prerequisite_skill_to_state':
```

Comment:
Normally we use this phrasing ("can do XXX") to convey something about the user's permissions, so perhaps you want to go with something like are_prerequisite_skills_editable instead.

Having said that, I think it would actually be a better idea to combine this and the above property into a single "is_exploration_whitelisted" variable, and use that instead throughout. /cc @brianrodri

Reply:
I actually think it's a good idea to expose it as a separate FE variable, it'll make changing things much simpler in the future and helps distinguish the fact that the two are separate but related.

Reply:
But they're really based on the same constant, right? I don't see them ever giving different results, so in that case shouldn't the branching happen in the frontend?

Reply:
Hi @aks681, I chatted with @brianrodri about this, and we suggest:

(a) Send a single value from the backend
(b) Use this to populate a service in the frontend that's dedicated to answering the question "is exploration curated" (or "is exploration part of story" or something). Not sure if this should be part of ExplorationContextService or something else.
(c) Any other frontend services that need this should use the service in (b) as a source of truth.

Reply:
Since all editor directives already import StateEditorService and it already stores whether the editor is a question or exploration editor, shall I return `is_exploration_whitelisted` from the backend and set a corresponding value in it?

Reply:
I think that sounds OK to me (on both counts).

Reply:
Ok, done.